[2025-01-23T04:17:32.350+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-23T04:17:32.364+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_process.dbt_seed scheduled__2025-01-22T00:00:00+00:00 [queued]>
[2025-01-23T04:17:32.369+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_process.dbt_seed scheduled__2025-01-22T00:00:00+00:00 [queued]>
[2025-01-23T04:17:32.370+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-23T04:17:32.381+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): dbt_seed> on 2025-01-22 00:00:00+00:00
[2025-01-23T04:17:32.384+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3715) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-23T04:17:32.386+0000] {standard_task_runner.py:72} INFO - Started process 3716 to run task
[2025-01-23T04:17:32.387+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dbt_process', 'dbt_seed', 'scheduled__2025-01-22T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/test_dbt_process.py', '--cfg-path', '/tmp/tmpf7s5m2i4']
[2025-01-23T04:17:32.388+0000] {standard_task_runner.py:105} INFO - Job 5: Subtask dbt_seed
[2025-01-23T04:17:32.430+0000] {task_command.py:467} INFO - Running <TaskInstance: dbt_process.dbt_seed scheduled__2025-01-22T00:00:00+00:00 [running]> on host c28716311b4b
[2025-01-23T04:17:32.496+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dbt_process' AIRFLOW_CTX_TASK_ID='dbt_seed' AIRFLOW_CTX_EXECUTION_DATE='2025-01-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-22T00:00:00+00:00'
[2025-01-23T04:17:32.497+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-23T04:17:32.503+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-23T04:17:32.504+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/***/include/dbt/northwind/ && dbt seed']
[2025-01-23T04:17:32.514+0000] {subprocess.py:99} INFO - Output:
[2025-01-23T04:17:34.678+0000] {subprocess.py:106} INFO - [0m04:17:34  Running with dbt=1.8.7
[2025-01-23T04:17:36.786+0000] {subprocess.py:106} INFO - [0m04:17:36  Registered adapter: snowflake=1.8.4
[2025-01-23T04:17:37.055+0000] {subprocess.py:106} INFO - [0m04:17:37  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-01-23T04:17:37.056+0000] {subprocess.py:106} INFO - There are 1 unused configuration paths:
[2025-01-23T04:17:37.056+0000] {subprocess.py:106} INFO - - models.northwind.example
[2025-01-23T04:17:37.170+0000] {subprocess.py:106} INFO - [0m04:17:37  Found 2 seeds, 1 model, 1 source, 459 macros
[2025-01-23T04:17:37.172+0000] {subprocess.py:106} INFO - [0m04:17:37
[2025-01-23T04:17:39.118+0000] {subprocess.py:106} INFO - [0m04:17:39  Concurrency: 200 threads (target='dev')
[2025-01-23T04:17:39.119+0000] {subprocess.py:106} INFO - [0m04:17:39
[2025-01-23T04:17:39.155+0000] {subprocess.py:106} INFO - [0m04:17:39  1 of 2 START seed file public.customer ......................................... [RUN]
[2025-01-23T04:17:39.168+0000] {subprocess.py:106} INFO - [0m04:17:39  2 of 2 START seed file public.products ......................................... [RUN]
[2025-01-23T04:17:40.874+0000] {subprocess.py:106} INFO - [0m04:17:40  2 of 2 OK loaded seed file public.products ..................................... [[32mINSERT 45[0m in 1.70s]
[2025-01-23T04:17:41.051+0000] {subprocess.py:106} INFO - [0m04:17:41  1 of 2 OK loaded seed file public.customer ..................................... [[32mINSERT 31[0m in 1.89s]
[2025-01-23T04:17:41.079+0000] {subprocess.py:106} INFO - [0m04:17:41
[2025-01-23T04:17:41.080+0000] {subprocess.py:106} INFO - [0m04:17:41  Finished running 2 seeds in 0 hours 0 minutes and 3.91 seconds (3.91s).
[2025-01-23T04:17:41.125+0000] {subprocess.py:106} INFO - [0m04:17:41
[2025-01-23T04:17:41.125+0000] {subprocess.py:106} INFO - [0m04:17:41  [32mCompleted successfully[0m
[2025-01-23T04:17:41.125+0000] {subprocess.py:106} INFO - [0m04:17:41
[2025-01-23T04:17:41.126+0000] {subprocess.py:106} INFO - [0m04:17:41  Done. PASS=2 WARN=0 ERROR=0 SKIP=0 TOTAL=2
[2025-01-23T04:17:42.792+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-23T04:17:42.828+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-23T04:17:42.828+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dbt_process, task_id=dbt_seed, run_id=scheduled__2025-01-22T00:00:00+00:00, execution_date=20250122T000000, start_date=20250123T041732, end_date=20250123T041742
[2025-01-23T04:17:42.878+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-23T04:17:42.897+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-23T04:17:42.899+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
